src/modules/endpoint/reducers/__tests__/switchLanguage.spec.js

```diff
@@ -270,14 +270,7 @@ describe('switchLangauge', () => {
   ]
 
   const initialState: CityContentStateType = {
```

Comment:
If you want you can rename this test to morthLanguage.spec.js

Reply:
Done.